capa/render/__init__.py

```diff
@@ -162,6 +162,40 @@ def convert_match_to_result_document(rules, capabilities, result):
     return doc
 
 
+def convert_meta_to_result_document(meta):
+    mbcs = meta.get("mbc", [])
+    meta["mbc"] = [parse_canonical_mbc(mbc) for mbc in mbcs]
+    return dict(meta)
```

Comment:
isn't `meta` already a `dict`? 😕

Reply:
i *think* it might also be a ruamel/yaml dict-like object, let me confirm.

Reply:
im not exactly sure. i agree, if no tests complain when removing this cast, then we should prefer the simpler code.